From d1ee2ff14ab1b5a37dc6091a874d721c352e93c1 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Fri, 26 May 2023 15:54:00 +0100
Subject: [PATCH 1/1] shim-systemd: Don't exit if we can't parse efivars

We can still install to the ESP and make use of shim's fallback to create
a dedicated UEFI entry
---
 src/bootloaders/shim-systemd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/bootloaders/shim-systemd.c b/src/bootloaders/shim-systemd.c
index 84605f3..c16f95f 100644
--- a/src/bootloaders/shim-systemd.c
+++ b/src/bootloaders/shim-systemd.c
@@ -347,7 +347,7 @@ static bool shim_systemd_init(const BootManager *manager)
 
         if (!boot_manager_is_image_mode((BootManager *)manager)) {
                 if (bootvar_init()) {
-                        return false;
+                        LOG_ERROR("Cannot parse EFI variables");
                 }
                 config.is_image_mode = 0;
         } else {
-- 
2.40.1

